scripts/aether_functions.sh

```diff
@@ -53,7 +53,11 @@ function build_ui_assets {
 function build_container {
     container=$1
     APP_REVISION=`git rev-parse --abbrev-ref HEAD`
-    APP_VERSION=`cat ./VERSION`
+    if [ ! -f VERSION ]; then
+        APP_VERSION="0.0.0"
```

Comment:
Back to `0.0.0` :joy:

Reply:
for build purposes only when `VERSION` isn't present

Reply:
I know it ;-)